third-party/CMakeLists.txt

```diff
@@ -87,8 +87,8 @@ endif()
 
 include(ExternalProject)
 
-set(LIBUV_URL https://github.com/libuv/libuv/archive/v1.9.1.tar.gz)
-set(LIBUV_SHA256 a6ca9f0648973d1463f46b495ce546ddcbe7cce2f04b32e802a15539e46c57ad)
+set(LIBUV_URL https://github.com/libuv/libuv/archive/v1.11.0.tar.gz)
+set(LIBUV_SHA256 6ec7eec6ecc24b1a8ffedebedb2fe9313fffb5410de89aaf784dd01080411c7a)
```

Comment:
Although the intention of equalsraf of the original commit is unknown, as far as I tried, ```jobstop( jobstart('cmd',{'pty':1}))```  in version 1.9.1 caused assert on line 159 of libuv/src/win/handle-intl.h.
I have not confirmed which version was fixed.